.github/workflows/test.yml

```diff
@@ -22,11 +22,19 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    - name: Build Stellar-Core Image
```

Comment:
❓ will these Build commands run 6 times? one for each of the networks and one for each options? And if so is there anyway to run only once? 🤔  (there might not be jw)

Reply:
We could cache the image layers probably, let me see. I know in the past GitHub Actions had a not great story for caching docker images within builds, but that might have changed. If I can't improve it, I don't think it's that big of a deal as they run in parallel, but it'd be nice if they could reuse prior layers though.

Reply:
I've addressed this in a separate PR, https://github.com/stellar/docker-stellar-core-horizon/pull/304, and once it is merged I'll update this one to incorporate it.